Essentials/src/main/java/com/earth2me/essentials/EssentialsPlayerListener.java

```diff
@@ -121,13 +121,26 @@ private static boolean isArrowPickupEvent() {
         }
     }
 
+    private static boolean isGameEventEvent() {
+        try {
+            Class.forName("org.bukkit.event.block.BlockReceiveGameEvent");
+            return true;
```

Comment:
Can we cache this result (e.g. from a static block) to avoid reflection constantly.

Reply:
only called during registration, once, on startup.